packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -150,6 +150,25 @@ describe('logs', () => {
     })
   })
 
+  describe('sampling', () => {
+    it('should be sampled when event bridge is present', () => {
+      updateExperimentalFeatures(['event-bridge'])
+      const sendSpy = spyOn(initEventBridgeStub(), 'send')
+
+      let configuration = { ...baseConfiguration, ...{ sampleRate: 0 } } as LogsInitConfiguration
+      let sendLog = originalStartLogs(configuration, new Logger(noop))
+      sendLog(DEFAULT_MESSAGE, {})
+
+      expect(sendSpy).not.toHaveBeenCalled()
+
+      configuration = { ...baseConfiguration, ...{ sampleRate: 100 } } as LogsInitConfiguration
```

Comment:
`baseConfiguration` already has sampleRate to 100 but I found it more expressive to assign it here.